app/views/shared/_popover.html.erb

```diff
@@ -0,0 +1,8 @@
+<%# available locals: message, placement %>
```

Comment:
nice to see this generic partial for popovers. I think would be worth to annotate it somewhere so we can re-use in other places too.